src/debugging-master/dashboard/styles.css

```
@@ -157,12 +161,58 @@ body {
+}
+
 .dbg-log-line__body {
     min-width: 0;
-    overflow: hidden;
     line-height: inherit;
 }
 
+.dbg-log-wrap {
+    white-space: pre-wrap;
+    overflow-wrap: anywhere;
+    word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)